packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -538,7 +558,7 @@ describe("Data Store Context Tests", () => {
                 assert.deepStrictEqual(gcData, emptyGCData, "GC data from getGCData should be empty.");
             });
 
-            it("can generate GC data with emtpy GC details in initial summary", async () => {
```

Comment:
Why was this test removed?

Reply:
That's because with this change, empty GC details and no GC details (previous test) are the same. The summarizer node will create a child with empty GC details if node exist.